.github/workflows/exclude-file.txt

```diff
@@ -11,4 +11,4 @@ config.site_url
 eks.amazonaws.com
 raw.githubusercontent.com
 creating-and-managing-clusters
-schema.js
\ No newline at end of file
+schema.js
```

Comment:
when did this get removed? 🤔

Reply:
It isn't removed, this changelist adds a newline after it. The diff is a little confusing.

Reply:
thanks makes sense 👍🏻